src/main/java/io/kestra/plugin/azure/eventhubs/Trigger.java

```diff
@@ -0,0 +1,155 @@
+package io.kestra.plugin.azure.eventhubs;
+
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.conditions.ConditionContext;
+import io.kestra.core.models.executions.Execution;
+import io.kestra.core.models.executions.ExecutionTrigger;
+import io.kestra.core.models.flows.State;
+import io.kestra.core.models.triggers.AbstractTrigger;
+import io.kestra.core.models.triggers.PollingTriggerInterface;
+import io.kestra.core.models.triggers.TriggerContext;
+import io.kestra.core.models.triggers.TriggerOutput;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.azure.eventhubs.serdes.Serdes;
+import io.kestra.plugin.azure.eventhubs.service.consumer.StartingPosition;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.Logger;
+
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * The {@link Trigger} can be used for triggering flow based on events received from Azure Event Hubs.
+ */
+@Plugin(examples = {
+    @Example(
+        title = "Trigger flow based on events received from Azure Event Hubs.",
+        full = true,
+        code = {
+            """
+                id: TriggerFromAzureEventHubs
+                namespace: company.team
+                tasks:
+                  - id: hello
+                    type: io.kestra.core.tasks.log.Log
+                    message: Hello there! I received {{trigger.eventsCount}} from Azure EventHubs!
+                triggers:
+                  - id: readFromEventHubs
+                    type: "io.kestra.plugin.azure.eventhubs.Trigger"
+                    interval: PT30S
+                    eventHubName: my-eventhub
+                    connectionString:"{{ secrets.azure.eventhubs.connectionString }}"
```

Comment:
```suggestion
                    connectionString:"{{ secret('EVENTHUBS_CONNECTION') }}"
```
Secret is a function that takes a key as parameter, please update all places